src/content/docs/network-performance-monitoring/monitoring-network-data/visualize-network-data.mdx

```diff
@@ -9,7 +9,9 @@ metaDescription: Visualize network performance data in New Relic One.
 
 After you've set up your network data for performance monitoring, you can go to New Relic One to install several custom curated visualizations designed specifically for network performance monitoring.
 
-Once you've set up these visualizations, you'll be able to start exploring the network data your devices are reporting to New Relic One.
+Once you've set up these visualizations, you'll be able to start exploring the network data your devices are reporting to New Relic One. For some examples of how you can group and monitor your switches, routers, etc. in dashboards so you can quickly aggregate, isolate, and analyze the health of your network, watch this short video (approx. 3:45 minutes).
```

Comment:
Is the second sentence too long?

Reply:
William Faulkner wrote it. I'll get Ernest Hemingway to edit it. :-)